userland/examples/ble_adv/main.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include "ble.h"
+#include <timer.h>
+
+int main(void)
+{
+  printf("\r\ndemo app\n");
+  unsigned char name[] = "TockOS";
+  unsigned char data[] = "1337";
+  unsigned char tx[] = {0x1};
+
+  printf("return for ble_adv_set_power %d\r\n", ble_adv_set_txpower(0));
+  printf("return for ble_adv_set_interval %d\r\n", ble_adv_set_interval(0));
```

Comment:
What does it mean to set the interval to zero (or negative, since this is an `int8_t`)? The valid advertising range is 20 ms to 10240 ms.